### DISCUSSION TITLE
Proxmox containers updates

### DISCUSSION THREAD

**Discussion by mind-drive** [tteck/Proxmox#955] [Ideas]
Hi Sir,

I'm having a hard time regarding the updates of a program for example uptime kuma in a container hosted by proxmox.
Is it possible that if there is an update for a particular system like uptime kuma that you will put a bash update command to update that program?

**Comment by tteck** [comment]
They're means to update, https://github.com/tteck/Proxmox/issues/914

![uptime](https://user-images.githubusercontent.com/21342670/213648001-e681f9ee-dfc2-4589-be64-2206a4a97b63.png)
```   __  __      __  _                   __ __                     
  / / / /___  / /_(_)___ ___  ___ v5  / //_/_  ______ ___  ____ _
 / / / / __ \/ __/ / __  __ \/ _ \   / ,< / / / / __  __ \/ __  /
/ /_/ / /_/ / /_/ / / / / / /  __/  / /| / /_/ / / / / / / /_/ / 
\____/ .___/\__/_/_/ /_/ /_/\___/  /_/ |_\__,_/_/ /_/ /_/\__,_/  
    /_/                                                          
 
 ✓ Stopped Uptime Kuma
 ✓ Pulled Uptime Kuma 1.19.6
 ✓ Updated Uptime Kuma
 ✓ Started Uptime Kuma
 ✓ Update Successfull
```
